### PR TITLE
add support for more languages

### PR DESCRIPTION
**Pull request by rleungx**
Add support for three languages:
- ABAP (.abap)
- COBOL (.cob, .cbl, .ccp, .cobol, .cpy)
- Groovy (.groovy, .grt, .gtpl, .gvy)

### CODE REVIEW

**Comment by Aaronepower** [comment]
@rleungx Thank you for this PR! I've left some comments on diff on some changes that need to be done before it can be merged.

File: languages.json [hunk 1]
```diff
@@ -1,5 +1,14 @@
 {
     "languages":{
+        "ABAP":{
```
Comment (Aaronepower) [inline languages.json]: This should be `Abap`, for changing the user facing name you can add the `"name": "ABAP"` property.

File: languages.json [hunk 2]
```diff
@@ -211,6 +220,18 @@
                 "cmakelists.txt"
             ]
         },
+        "COBOL": {
```
Comment (Aaronepower) [inline languages.json]: Same as above.

**Comment by rleungx** [comment]
Done. @Aaronepower

**Comment by Aaronepower** [comment]
@rleungx Thanks agsin for this PR and congrats on your first contribution! Which is also the 200th pull request!

**Pull request merged**